examples/with-less/razzle.config.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const lessOption = {
+  javascriptEnabled: true,
+  modifyVars: { '@info-color': '#1DA57A' },
+};
+
+module.exports = {
+  plugins: [
+    {
+      name: 'less',
+      options: {
+        less: {
+          dev: Object.assign(lessOption, {
+            sourceMap: true,
+          }),
+          prod: Object.assign(lessOption, {
+            sourceMap: false,
+          }),
+        },
+      },
+    },
+  ],
+};
```

Comment:
just use the default config to keep examples as minimal as possible
```suggestion
module.exports = {
  plugins: ['less'],
};
```

Reply:
Changed according to suggestion. Thank you.